guidelines/guidelines-de.php

```diff
@@ -0,0 +1,45 @@
+<?php
+use yii\helpers\Url;
+
+$this->title = Yii::t('app', 'Richtlinien');
+?>
+<h1>
+    <?= $this->title ?>
+</h1>
+<p class="has-text-right is-italic">
+    Zuletzt geändert: 2020-04-10
+</p>
+<p>
+    Um eine sichere Gemeinschaft zu erhalten, gibt es einige Rechtlinien, die von dir befolgt werden müssen <br>
+    Durch die Verwendung von DISBOARD erklärst du dich mit folgenden Richtlinien einverstanden:
+</p>
+<ul>
+    <li>
+        Die Verwendung von Schimpfwörtern oder NSFW (sexueller Inhalt) nach unserem Ermessen in der Meta (Titel, Beschreibung und Bild), Rezension oder anderen Inhalten eines Servers, die von anderen Benutzern auf DISBOARD gesehen werden können, ist nicht erlaubt. Dies führt zur Entfernung des Inhalts, du hast jedoch die Möglichkeit, den Inhalt in einer angemessenen Sprache neu zu posten.
+    </li>
+    <li>
+        Du darfst keine Gesetze oder Vorschriften in deinem Aufenthaltsland verletzen oder deren Verletzung fördern.
+    </li>
+    <li>
+        Server, die gegen die <a href="https://discordapp.com/guidelines" target="_blank">Discord Community Richtlinien</a> verletzen, sind nicht erlaubt.
```

Comment:
discordapp.com should be changed to discord.com

Reply:
Fixed, thank you. :)